docs/Containerfile.5.md

```diff
@@ -126,6 +126,8 @@ Current supported mount TYPES are bind, cache, secret and tmpfs.
 
               . bind-nonrecursive: do not setup a recursive bind mount.  By default it is recursive.
 
+              . from: Image name for the root of the source. Defaults to the build context.
```

Comment:
Typo: "." instead of "·"
"Image" shouldn't be capitalized, for consistency with the descriptions of "bind-propagation" and "bind-nonrecursive".